in-adapter-api/src/main/java/gohigher/support/auth/RefreshTokenCookieProvider.java

```diff
@@ -34,4 +39,20 @@ private ResponseCookie.ResponseCookieBuilder createTokenCookieBuilder(String val
 			.path(tokenRequestUri)
 			.sameSite(SameSite.NONE.attributeValue());
 	}
+
+	public String extractToken(final Cookie[] cookies) {
+		validateCookiesNotEmpty(cookies);
+		final Cookie cookie = Arrays.stream(cookies)
+			.filter(it -> it.getName().equals(REFRESH_TOKEN_KEY))
+			.findAny()
+			.orElseThrow(() -> new GoHigherException(AuthErrorCode.EMPTY_REFRESH_TOKEN_COOKIE));
+
+		return cookie.getValue();
+	}
```

Comment:
```suggestion
	public String extractToken(Cookie[] cookies) {
		validateCookiesNotEmpty(cookies);
		return Arrays.stream(cookies)
			.filter(it -> it.getName().equals(REFRESH_TOKEN_KEY))
			.findAny()
			.orElseThrow(() -> new GoHigherException(AuthErrorCode.EMPTY_REFRESH_TOKEN_COOKIE))
			.getValue();
	}
```
stream 에서 cookie 로 할당한 후 getter 를 호출한건 해당 값이 쿠키이며 쿠키에서 원하는 값을 가져온다고 명시하기 위해서일까요?? ealry return 이 가능할 것 같아서요!!

Reply:
앗 반영할게요ㅎㅎ